examples/lb3-application/src/controllers/note.controller.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  post,
+  param,
+  get,
+  getFilterSchemaFor,
+  getModelSchemaRef,
+  getWhereSchemaFor,
+  patch,
+  put,
+  del,
+  requestBody,
+} from '@loopback/rest';
+import {Note} from '../models';
+import {NoteRepository} from '../repositories';
+
+export class NoteController {
```

Comment:
I see that you are adding a LB4 controller to `lb3-application`. What's your intention here? Originally, `lb3-application` was showing the outcome of migration of LB3 app to LB4.

The same comment applies to the memory datasource, Note model and repository.

Reply:
We now have to look at the example app as an Express app which loads LB4 and LB3 models. There were tests to ensure LB3 models are loaded, we should ensure LB4 models are loaded too.

As long as we don't include these additions in the guide, I think it is alright for them to be there, and actually good to have them because they will help discover any potential issues in future. Besides, users will get to see LB3 and LB4 models working together right out of the box.

Reply:
Removed them.